### PR TITLE
Cutomize for Nov 2020 workshop agenda.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: workshop/README.md [hunk 1]
```diff
@@ -80,7 +80,6 @@ This file has the following attributes:
 | [Machine Learning with AutoAI](machine-learning-autoai/README.md) | Use AutoAi to quickly generate a Machine Learning pipeline and model |
 | [Deploy and Test Machine Learning Models](machine-learning-deployment-scoring/README.md) | Deploy and machine learning models using several approaches |
 | [Monitoring models with OpenScale GUI (Fastpath Monitoring)](openscale-fastpath/README.md) | Quickly deploy an OpenScale demo with FastPath |
-| [Monitoring models with OpenScale (Notebook)](openscale-notebook/README.md) | See the OpenScale APIs in a Jupyter notebook and manually configure the monitors |
```
Comment (jrtorres) [inline workshop/README.md:83]: @scottdangelo We want to showcase the manual config (openscale-manual-config) this time around, no ?

File: workshop/SUMMARY.md [hunk 2]
```diff
@@ -12,7 +12,6 @@
 * [Machine Learning with AutoAI](machine-learning-autoai/README.md)
 * [Deploy and Test Machine Learning Models](machine-learning-deployment-scoring/README.md)
 * [Monitoring models with OpenScale GUI (Fastpath Monitoring)](openscale-fastpath/README.md)
-* [(Optional) Monitoring models with OpenScale (Notebook)](openscale-notebook/README.md)
```
Comment (jrtorres) [inline workshop/SUMMARY.md:15]: @scottdangelo We want to showcase the manual config (openscale-manual-config) this time around, no ?

**Review by jrtorres — CHANGES_REQUESTED** [review]
I think we want to use the manual config instead of full config notebook.

File: workshop/README.md (hunk 1 above)
Comment (scottdangelo) [reply workshop/README.md:83]: OK, I wasn't sure. Do you want me to delete FastPath, or do both?

File: workshop/SUMMARY.md (hunk 2 above)
Comment (scottdangelo) [reply workshop/SUMMARY.md:15]: I'll add the manual config. Should I delete the fastpath?

File: workshop/README.md (hunk 1 above)
Comment (jrtorres) [reply workshop/README.md:83]: I would keep both.

File: workshop/SUMMARY.md (hunk 2 above)
Comment (jrtorres) [reply workshop/SUMMARY.md:15]: I would keep both.

**Review by jrtorres — APPROVED** [review]

**Pull request merged**